content/about.html

```diff
@@ -1,7 +1,7 @@
 <html devsite>
 
 <head>
-  <title>What is web.dev?</title>
+  <title>Where you can learn & measure</title>
```

Comment:
I really like the wording of this new title. The wrapping text is a little off-putting, but then it looks OK across device widths, and the same pattern applies to the landing page (wrapping text in title). So I'm gonna go with keeping this new title-- as I love how it aligns with the top nav!